index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    
+    <h2 id="title">Survey Form</h2>
+     
+    <p id="description">
+      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Qui exercitationem dolore natus, earum, nesciunt ad consequuntur corporis repellendus nisi, eligendi quos voluptatibus harum iusto dignissimos? Sed deserunt omnis enim dolore.
+    </p>
+    
+    <form action="" id="survey-form">
+     <label id="name-label">Enter Your Name:</label>
+     <input type="text" id="name" name="username" placeholder="Enter Name" required /> <br/>
+     <label id="email-label">Enter Your Email Address</label>
+     <input type="email" name="email" id="email" placeholder="Enter Email" required /> <br/>
+     <label id="number-label">Enter Number (0-100)</label>
+     <input type="number" name="number" id="number" max="100" min="0" placeholder="Enter Number" /> <br/>
```

Comment:
Please consider adding a `for` attribute to all the labels on this form, the value of the `for` attribute should match the `id` attribute of the `input` element it's labeling. e.g 
```HTML
<label for="username" id="name-label">Enter your name</label>
<input type="text" id="name" name="username" placeholder="Enter Name" required" />
```
Matching labels to input elements makes it easy for people using web accessibility tools to navigate web forms easily.